jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -255,11 +305,77 @@ public void toyNullTestAPI() throws Exception {
         "}");
   }
 
+  private String getToyTestSrcWithExpectAnnotations() {
+    return "package toys;"
```

Comment:
This code is weird to me. Should probably be multiple `addInputSource`: one for each interface and one for the class. Java won't let you define multiple top-level types in the same file, as far as I know. Although it seems it does work in the code below, um...

Reply:
Removed the test.